tree/tree.go

```diff
@@ -117,6 +116,19 @@ func splitNameAndLength(input string) (string, float64, bool, error) {
 	return "", 0, false, fmt.Errorf("Error: %s should only have one or two colons\n", input)
 }
 
+func ReadMultiLineTree(input string) string {
```

Comment:
I think this also handles single line tree files, right?  I think this should actually replace the old version of ReadNewick, rather than become its own function.  I think this because it more improves something that already exists.  That way we don't end up with two read functions for trees that we need to maintain.

Reply:
I think moving this to the fileio package makes sense and then calling it in readNewick makes the most sense.

Reply:
You can move this to fileio and name it with a general name like ReadFileToSingleLineString, but I would also update ReadNewick so that it calls your newly created fileio.ReadFileToSingleLineString and therefore handles both multi-line and single-line trees.

Reply:
Yes, just to clarify though, these functions are identical (the tree.go and expandedTree.go functions) so I think I'm going to keep the general option and call the new fileio general function in both cases.